api/libproject/generate_appengine_cron_yaml.rb

```diff
@@ -9,12 +9,15 @@
 
 env = ARGV[0]
```

Comment:
I'd like to use it more consistently, per this comment: https://github.com/all-of-us/workbench/blob/a673cf0e8faa04bf33a4b15100ae2c2fb566705a/api/libproject/devstart.rb#L36

Most of our commands use "project", which doesn't accommodate "local" very well, as side-effect in several cases we have forked commands between local/cloud. Renamed to `aou_env`

Reply:
Sounds good.

It would be worth clarifying the taxonomy eventually. I was surprised in the auditing work that local and test events went to the same place, but then I realized they're the same Logging namespace. I think there are lots of places where it's easy to get confused.